www/Lib/load_database.php

```diff
@@ -5,20 +5,31 @@
 error_reporting(E_ALL);
 ini_set('display_errors', 'on');
 
-require "settings.php";
-
-if (!isset($settings["sql"])) {
-    echo "No sql settings found in settings.php<br />";
-    die();
+if ( file_exists("settings.php") ){
```

Comment:
You may even want to switch this around so env vars win if they are specified even if `settings.php` exists. ofc that's a smidge tricky because you'd end up checking for a canary env var or even more grisly, check for all of them.